### PR TITLE
chore(#84): documented callback url requirements.

### PR DESCRIPTION
**Pull request by samuelimoisili**
# Description

Added documentation for the FHIR callback requirement.

Closes medic/interoperability#84

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: WORKFLOW.md [hunk 1]
```diff
@@ -145,6 +145,8 @@ In the LTFU workflow, the `Endpoint` is crucial in creating a `ServiceRequest`.
 - **ENDPOINT_IDENTIFIER:** An identifier for the `Endpoint` that can be used when querying the FHIR database in the future.
 - **ORG_CALLBACK_URL:** A callback URL that the mediator can use to contact the requesting system (`Organization`) in the future when a `ServiceRequest` has been fulfilled.
 
+> **NOTE** The FHIR Subscription created in the future requires `ORG_CALLBACK_URL` to accept HTTP PUT requests matching this path `${ORG_CALLBACK_URL}/:resourceType/:resourceId` and return a 200. In this workflow, the callback should expect to receive an `Encounter` resource sent back to the requesting system on `${ORG_CALLBACK_URL}/Encounter/:id`.
```
Comment (andrablaj) [inline WORKFLOW.md:148]:
```suggestion
> **NOTE** The FHIR `Subscription` that will be created ulteriorly requires `ORG_CALLBACK_URL` to accept HTTP `PUT` requests matching this path `${ORG_CALLBACK_URL}/:resourceType/:resourceId` and return a 200. In this workflow, the callback should expect to receive an `Encounter` resource sent back to the requesting system on `${ORG_CALLBACK_URL}/Encounter/:id`.
```

**Review by andrablaj — CHANGES_REQUESTED** [review]
Tiny tweak on the style, but it looks good otherwise!

**Review by andrablaj — APPROVED** [review]
👌

**Review by lorerod — COMMENTED** [review]
Thanks!

**Pull request merged**